akka-typed-tests/src/test/scala/akka/typed/persistence/scaladsl/PersistentActorCompileOnlyTest.scala

```diff
@@ -305,32 +306,37 @@ object PersistentActorCompileOnlyTest {
       PersistentActor.immutable[Command, Event, List[Id]](
         persistenceId = "basket-1",
         initialState = Nil,
-        actions =
-          Actions.byState(state ⇒
-            if (isFullyHydrated(basket, state)) Actions { (ctx, cmd, state) ⇒
+        commandHandler =
+          CommandHandler.byState(state ⇒
+            if (isFullyHydrated(basket, state)) CommandHandler { (ctx, cmd, state) ⇒
               cmd match {
                 case AddItem(id)    ⇒ addItem(id, ctx.self)
-                case RemoveItem(id) ⇒ Persist(ItemRemoved(id))
+                case RemoveItem(id) ⇒ Effect.persist(ItemRemoved(id))
                 case GotMetaData(data) ⇒
-                  basket = basket.updatedWith(data); PersistNothing()
-                case GetTotalPrice(sender) ⇒ sender ! basket.items.map(_.price).sum; PersistNothing()
+                  basket = basket.updatedWith(data)
+                  Effect.done
+                case GetTotalPrice(sender) ⇒
+                  sender ! basket.items.map(_.price).sum
+                  Effect.done
               }
             }
-            else Actions { (ctx, cmd, state) ⇒
+            else CommandHandler { (ctx, cmd, state) ⇒
               cmd match {
                 case AddItem(id)    ⇒ addItem(id, ctx.self)
-                case RemoveItem(id) ⇒ Persist(ItemRemoved(id))
+                case RemoveItem(id) ⇒ Effect.persist(ItemRemoved(id))
                 case GotMetaData(data) ⇒
                   basket = basket.updatedWith(data)
                   if (isFullyHydrated(basket, state)) {
                     stash.foreach(ctx.self ! _)
```

Comment:
Not sure I know the context for this example, but using stash like this doesn't look good. The messages are placed last in the mailbox and original order is not preserved.

Reply:
Yeah, not quite sure I understand what it demonstrates even.